src/olympia/addons/admin.py

```diff
@@ -194,9 +194,22 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
                     url += '?' + request.GET.urlencode()
                 return http.HttpResponsePermanentRedirect(url)
 
-        return super(AddonAdmin, self).change_view(
-            request, object_id, form_url, extra_context=None,
+        response = super(AddonAdmin, self).change_view(
```

Comment:
nit: we don't need the class name in `super().` in modern python versions, so it's okay to strip it out when touching code that does it the old way.